src/components/WebComponentProject/WebComponentProject.jsx

```diff
@@ -2,7 +2,6 @@ import React, { useEffect, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { useMediaQuery } from "react-responsive";
 
-import "../../assets/stylesheets/EmbeddedViewer.scss";
```

Comment:
💭 I _think_ this needs re-adding. I'm not sure what styles it pulls out but the `WebComponentProject` is definitely used so worth keeping these styles in for now